nixos/modules/services/monitoring/munin.nix

```diff
@@ -193,6 +193,9 @@ in
       };
     };
 
+    # munin_stats plugin breaks as of 2.0.33 when this doesn't exist
+    systemd.tmpfiles.rules = [ "d /var/run/munin 0755 munin munin -" ];
```

Comment:
Perhaps "serviceConfig.RuntimeDirectory=munin" is more appropriate here?

Reply:
Probably.

Reply:
I was totally not aware of those serviceConfig options, they look great, thank you! 😃

Problem here would be though that `/var/run/munin` is not only used as your usual pid file folder, but also the plugin state is redirected from the default location of `/var/lib/munin/plugin-state` to `/var/run/munin` via the MUNIN_PLUGSTATE variable – the latter causing the actual problem with munin_stats plugin. It might not be good to clear the plugin state on every restart of the service. But I don't know why that's actually done.

The dual use traces back to the first commit of the service, @domenkozar, do you still know your reasons why you set `MUNIN_PLUGSTATE` to `/var/run/munin`?